### ISSUE TITLE
Using camera feature in mobile  responsive way is not working

### ISSUE THREAD

**Issue by chetna-webonise** [elninotech/uppload#232]
I have integrated  uppload plugin in the React webosite ,it works fine for all the types of services including camera.
But when I try to upload image using camera option in mobile for the same site, the camera option does not shows up.

**Comment by AnandChowdhary** [comment]
Yes, based on whether the mobile device supports using Camera, the camera option may not appear. Chances are you're using iOS?

This is the `supports` expression (https://github.com/elninotech/uppload/blob/master/src/services/camera.ts#L20):

```ts
const supports = !!(
      window.navigator.mediaDevices &&
      window.navigator.mediaDevices.enumerateDevices &&
      !/iPhone|iPad|iPod|Android/i.test(navigator.userAgent)
    );
```

**Comment by AnandChowdhary** [comment]
It seems like it's hardcoded to make sure mobile devices don't show the camera option, probably because of the formerly poor support of `window.navigator.mediaDevices`. I'll make sure I do some testing and remove that last condition.

**Comment by AnandChowdhary** [comment]
However, most mobile devices support clicking a photo from the camera using the Local (file upload) service.

**Comment by chetna-webonise** [comment]
Thanks for your quick response @AnandChowdhary

**Comment by chetna-webonise** [comment]
My webiste  uses **uppload** plugin  for editing profile pictures. The feature i m working on has to be compatible with both web and mobile view,However for webiste it works fantastic, but when we switch to mobile the camera option disappears.
I don't want to add any other overhead on the code by writing extra functionality for mobile device to use camera and then take  picture and upload.
Since your plugin does too good by giving all edit options..

Hope to listen from you soon on this.

**Comment by chetna-webonise** [comment]
> However, most mobile devices support clicking a photo from the camera using the Local (file upload) service.

Yes it does. 
Is the text  **drop fies here and select file button** customisable , there we can add both options like **take a photo or select a file**  fro mobile view , so the user does not get confused.
